api/runes/tests/api/configurable-limit.test.ts

```diff
@@ -0,0 +1,404 @@
+/**
+ * @jest-environment node
+ */
+
+// Set environment variables before any modules are imported
+process.env.RUNES_PGHOST = process.env.RUNES_PGHOST || 'localhost';
+process.env.RUNES_PGPORT = process.env.RUNES_PGPORT || '5432';
+process.env.RUNES_PGUSER = process.env.RUNES_PGUSER || 'test';
+process.env.RUNES_PGPASSWORD = process.env.RUNES_PGPASSWORD || 'test';
+process.env.RUNES_PGDATABASE = process.env.RUNES_PGDATABASE || 'test';
+process.env.API_RESULTS_MAX_LIMIT = process.env.API_RESULTS_MAX_LIMIT || '100';
```

Comment:
move these to `beforeEach` or you can put them in `beforeAll` if you want to do it only once